circuit_knitting/cutting/__init__.py

```diff
@@ -87,6 +104,7 @@
 from .wire_cutting_transforms import cut_wires, expand_observables
 
 __all__ = [
+    "find_cuts",
```

Comment:
This line actually ends up referencing the `find_cuts` module, not the `find_cuts` function.

```
>>> from circuit_knitting.cutting import *
>>> find_cuts
<module 'circuit_knitting.cutting.find_cuts' from '/home/garrison/serverless/circuit-knitting-toolbox7/circuit_knitting/cutting/find_cuts.py'>
>>> find_cuts.find_cuts
<function find_cuts at 0x7f2ee27ed1c0>
```

I think my preference would be to expose the `find_cuts` function instead, and possibly also the two classes that are meant to be used from it, right here in `__init__.py`.  We might even want to rename the filename `find_cuts.py` so we don't have two different things (a module and a function) with exactly the same name at the same level of the hierarchy.

Reply:
Yes, `cut_finding.py` doesnt work because there is a package at this level, I didn't consider the function name inside the module being a problem

Reply:
Renamed the module to `automated_cut_finding`, added the settings classes to the `cutting` package, and updated the notebook+test imports